rllib/utils/minibatch_utils.py

```diff
@@ -1,5 +1,6 @@
 import math
 
+
```

Comment:
two empty lines?

Reply:
I can get rid of these, they're left over from previous commits